assets/src/components/app.tsx

```diff
@@ -1,78 +1,95 @@
         />
-      )}
+      </svg>
     </div>
-  )
-}
+    <ul>
+      <li>
+        <NavLink
+          activeClassName="m-tab-bar__link--active"
+          className="m-tab-bar__link"
+          exact={true}
+          title="Routes"
+          to="/"
+        >
+          <svg
+            viewBox="0 0 48 48"
+            width="24"
+            height="24"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              className="m-tab-bar__icon"
+              d="m32.73 39.86v-12.08a4.37 4.37 0 0 1 0-7.56v-12.08a4.36 4.36 0 1 1 4.36 0v12.08a4.37 4.37 0 0 1 0 7.56v12.08a4.36 4.36 0 1 1 -4.36 0zm-17.46 0a4.36 4.36 0 1 1 -4.36 0v-12.08a4.37 4.37 0 0 1 0-7.56v-12.08a4.36 4.36 0 1 1 4.36 0v12.08a4.37 4.37 0 0 1 0 7.56z"
+            />
+          </svg>
+        </NavLink>
+      </li>
+      <li>
+        <NavLink
+          activeClassName="m-tab-bar__link--active"
+          className="m-tab-bar__link"
+          title="FAQ"
+          to="/faq"
+        >
+          <svg
+            viewBox="0 0 48 48"
+            width="24"
+            height="24"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              className="m-tab-bar__icon"
+              d="m24 0a24 24 0 1 0 24 24 24 24 0 0 0 -24-24zm1.71 40.05a3.5 3.5 0 0 1 -4.82 0 3.15 3.15 0 0 1 -1-2.36 3.23 3.23 0 0 1 1-2.38 3.48 3.48 0 0 1 4.82 0 3.24 3.24 0 0 1 1 2.38 3.16 3.16 0 0 1 -1 2.36zm7-20.46a9.55 9.55 0 0 1 -2 2.59c-.79.73-1.57 1.47-2.36 2.22a10 10 0 0 0 -2 2.64 8.43 8.43 0 0 0 -.76 3.96h-4.59a14.32 14.32 0 0 1 .55-4.3 9.62 9.62 0 0 1 1.41-2.91 13.55 13.55 0 0 1 1.84-2c.66-.59 1.27-1.15 1.85-1.7a7.19 7.19 0 0 0 1.35-1.9 5 5 0 0 0 .55-2.45 4.58 4.58 0 0 0 -1.21-3.39 4.45 4.45 0 0 0 -3.27-1.2 4.91 4.91 0 0 0 -3.07 1.12 4.42 4.42 0 0 0 -1.46 3.73h-5.07a8.89 8.89 0 0 1 1.29-5 8.12 8.12 0 0 1 3.47-3 11.37 11.37 0 0 1 4.88-1 9.55 9.55 0 0 1 6.89 2.4 8.47 8.47 0 0 1 2.49 6.38 8 8 0 0 1 -.76 3.81z"
```

Comment:
All the SVG data makes it harder to see the big picture of how this component works. What do you think about pulling each SVG into a function?

We also have `icon.tsx` that loads icons from files, though that's probably not necessary to hook into here.